addOns/client/src/main/resources/org/zaproxy/addon/client/resources/Messages.properties

```diff
@@ -8,8 +8,16 @@ client.components.type.input = Input
 client.components.type.link = Link
 
 client.desc = Client Side Integration
+client.details.popup.copy.hrefs = Copy HREFs to Clipboard
+client.details.popup.copy.ids = Copy IDs to Clipboard
+client.details.popup.copy.texts = Copy Texts to Clipboard
 client.details.title = Client Details
 client.error.nofile = File not found : {0}
+client.history.popup.copy.nodeids = Copy Node IDs to Clipboard
+client.history.popup.copy.nodenames = Copy Node Names to Clipboard
+client.history.popup.copy.texts = Copy Texts to Clipboard
+client.history.popup.copy.types = Copy Types to Clipboard
+client.history.popup.copy.urls = Copy Source URLs to Clipboard
```

Comment:
For the help too, do we really need to say "to Clipboard"?

Reply:
Thats what we do in the core, so for consistancy "yes" 😁